WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginUsernamePasswordFragment.java

```diff
@@ -0,0 +1,457 @@
+            public void onClick(View v) {
+                next();
+            }
+        });
+
+        mUsernameEditText.setOnEditorActionListener(new TextView.OnEditorActionListener() {
+            @Override
+            public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
+                if (event != null
+                        && event.getAction() == KeyEvent.ACTION_UP
+                        && event.getKeyCode() == KeyEvent.KEYCODE_ENTER) {
+                    mPasswordEditText.requestFocus();
+                }
+
+                // always consume the event so the focus stays in the EditText
+                return true;
+            }
+        });
+
+        mPasswordEditText.setOnEditorActionListener(new TextView.OnEditorActionListener() {
+            @Override
+            public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
+                if (event != null
+                        && event.getAction() == KeyEvent.ACTION_UP
+                        && event.getKeyCode() == KeyEvent.KEYCODE_ENTER) {
+                    next();
+                }
+
+                // always consume the event so the focus stays in the EditText
+                return true;
+            }
+        });
+
+        rootView.findViewById(R.id.login_lost_password).setOnClickListener(new OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (mLoginListener != null) {
+                    mLoginListener.forgotPassword();
+                }
+            }
+        });
+
+        return rootView;
+    }
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        Toolbar toolbar = (Toolbar) view.findViewById(R.id.toolbar);
+        ((AppCompatActivity) getActivity()).setSupportActionBar(toolbar);
+
+        ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setDisplayShowTitleEnabled(false);
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
+
+        if (savedInstanceState == null) {
+            EditTextUtils.showSoftInput(mUsernameEditText);
+        }
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        if (savedInstanceState != null) {
+            mInProgress = savedInstanceState.getBoolean(KEY_IN_PROGRESS);
+            mLoginFinished = savedInstanceState.getBoolean(KEY_LOGIN_FINISHED);
+
+            mRequestedUsername = savedInstanceState.getString(KEY_REQUESTED_USERNAME);
+            mRequestedPassword = savedInstanceState.getString(KEY_REQUESTED_PASSWORD);
+
+            if (mInProgress) {
+                showProgressDialog();
+            }
+        }
+    }
+
+    @Override
+    public void onAttach(Context context) {
```

Comment:
`LoginUsernamePasswordFragment` extends the support lib's Fragment class so, `onAttach(Context context)` is the variant called, even for older API versions. The `onAttach(Activity activity)` variant is already deprecated for the version we're using. I think there is no need to override the Activity variant, right?